lib/v2/gov/cmse/index.js

```diff
@@ -0,0 +1,67 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    ctx.path = ctx.path.replace(/(^\/cmse|\/$)/g, '');
+
+    const rootUrl = 'http://www.cmse.gov.cn';
+    const currentUrl = `${rootUrl}${ctx.path === '' ? '/xwzx/zhxw/' : `${ctx.path}/`}`;
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+
+    const $ = cheerio.load(response.data);
+
+    let items = $('#list li a')
+        .slice(0, ctx.query.limit ? parseInt(ctx.query.limit) : 15)
+        .toArray()
+        .map((item) => {
+            item = $(item);
+
+            const pubDate = item.next().text();
+            const link = new URL(item.attr('href'), currentUrl);
```

Comment:
```suggestion
            const link = new URL(item.attr('href'), currentUrl).href;
```
link should be a `String` instead of `URL` for ctx.cache.tryGet(...)

Reply:
👍